package.json

```diff
@@ -180,6 +176,9 @@
     "xml-writer": "1.7.0"
   },
   "devDependencies": {
+    "@babel/core": "7.1.2",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
Needed for Jest.
See https://jestjs.io/docs/en/getting-started.html#using-babel